Packages/io.chainsafe.web3-unity.hyperplay/package.json

```diff
@@ -0,0 +1,33 @@
+{
+    "name": "io.chainsafe.web3-unity.hyperplay",
+    "version": "1.0.0",
+    "displayName": "web3.unity SDK HyperPlay",
+    "description": "This package includes the integration of HyperPlay, which you can use with our Gaming SDK.",
+    "license": "LGPL-3.0-only",
+    "licensesUrl": "https://github.com/ChainSafe/web3.unity/blob/main/LICENSE",
+    "documentationUrl": "https://docs.gaming.chainsafe.io/",
+    "dependencies": {
+        "com.unity.nuget.newtonsoft-json": "3.0.2",
+        "io.chainsafe.web3-unity": "2.5.0"
```

Comment:
let's up this to 2.6 since final package will be 2.6

Reply:
done